modes/tmtv/src/index.ts

```diff
@@ -120,7 +120,7 @@ function modeFactory({ modeConfiguration }) {
             minValue: 'Minimum Value',
             maxValue: 'Maximum Value',
             meanValue: 'Mean Value',
-            volume: 'Volume',
+            volume: 'Volume (ml)',
```

Comment:
This is supposed to display as part of the value.  Properly it would be a fix in CS3D as to how it generates the values, in order to allow other volume units such as ul for microscopy.